### PR TITLE
Use one global string for "CoinJoin"

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

File: src/util.cpp [hunk 1]
```diff
@@ -90,6 +90,8 @@ const int64_t nStartupTime = GetTime();
 //Dash only features
 bool fMasternodeMode = false;
 bool fDisableGovernance = false;
+std::string gCoinJoinName = "CoinJoin";
```
Comment (kxcd) [inline src/util.cpp:93]: Thanks for this PR!  Does it make sense to use the `const` keyword here?

Comment (PastaPastaPasta) [reply src/util.cpp:93]: Yes, this should be const.

**Review by PastaPastaPasta — CHANGES_REQUESTED** [review]
Concept ACK, please make this const though

**Comment by UdjinM6** [comment]
Done

**Comment by PastaPastaPasta** [comment]
Needs rebase

**Review by PastaPastaPasta — APPROVED** [review]
utACK

**Pull request merged**